.github/workflows/functional-test.yaml

```diff
@@ -71,15 +70,15 @@ env:
   ACTION_LINK: '${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}'
   # Server where terraform test modules are deployed
   TF_RECIPE_MODULE_SERVER_URL: "http://tf-module-server.radius-test-tf-module-server.svc.cluster.local"
+  # The rad CLI download URL
+  RAD_CLI_URL: https://get.radapp.dev/tools/rad/install.sh
```

Comment:
This is unused (as it should be). We want to make sure we're using the version of the CLI from the PR.